DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -215,33 +200,42 @@ constexpr T String2(const char (&str)[N])
 /// have to be studied.
 template <int N, typename PrinterPolicy = Internal::defaultPrinter>
 struct Descriptor {
-  static_assert(Internal::NumberOfActiveBits<N>::value == 1 && N <= 8,
+  static_assert(Internal::NumberOfActiveBits<N>::value == 1,
 		"size is required to be 1 or a multiple of 2");
```

Comment:
Should this not read: `"size is required to be a positive power of 2"`?, i.e:
```c++
N > 0 && (N & (N - 1)) == 0
```

or, the original statement:
```c++
N > 0 && (N % 2) == 0
```